rust/processor/Cargo.lock

```diff
@@ -1,2382 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
why the deletion btw?

Reply:
oh this hsould'been added back. it was causing merge conflict.  will add back